design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
+    ): UpdatedPricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    assignBillingAddressToNegotiableQuote(
+        billingAddress: NegotiableQuoteBillingAddressInput!
+    ): AssignBillingAddressToNegotiableQuoteOutput @doc(description:"Assigns new billing address to negotiable quote")
+
+    estimateShippingCostForNegotiableQuote(
+        shippingAddress: NegotiableQuoteEstimateShippinAddressInput!
+    ): EstimateShippingCostForNegotiableQuoteOutput @doc(description:"Estimate shipping costs for a negotiable quote")
+
+    estimateShippingCostByAddressId(
+        shippingAddressId: NegotiableQuoteEstimateShippinAddressIdInput!
+    ): EstimateShippingCostByAddressIdOutput @doc(description:"Estimate shipping costs for a negotiable quote by Address ID")
+
+    estimateShippingInformationForNegotiableQuote(
+        addressInformation: NegotiableQuoteShippingInformationInput!
+    ): EstimateShippingInformationForNegotiableQuoteOutput @doc(description:"Get list of payment options and the order totals by address and shipping info")
+
+    setShippingMethodToNegotiableQuote(
+        negotiableQuoteShippingMethod: NegotiableQuoteShippingMethodInput!
+    ): setShippingMethodToNegotiableQuoteOutput @doc(description:"Set negotiable quote shipping method")
+
+    setCouponCodeToNegotiableQuote(
+        negotiableQuoteCouponCode: NegotiableQuoteCouponCodeInput!
```

Comment:
How these mutations are different from `applyCouponToCart` and `removeCouponFromCart`? They both accept `cart_id` as an input.

Reply:
As you said both of them operates on Cart and they are now way different, we removed these mutations